packages/context/src/lib/context-manager/shared/context.service.ts

```diff
@@ -493,7 +493,8 @@ export class ContextService {
           title: layer.options.title,
           zIndex: ++i,
           visible: layer.visible,
-          security: layer.security
+          security: layer.security,
+          opacity: layer.opacity
         } as LayerOptions,
```

Comment:
Les `as` est un peu comme le `any`, on doit l'éviter parce qu'il force un typage sans pour autant respecter l'interface. Au lieu du `as`, on peut utiliser [l'opérateur satisfies](https://typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#the-satisfies-operator)